src/stores/FilterableTableStore.js

```diff
@@ -0,0 +1,8 @@
+export class FilterableTableStore {
```

Comment:
No, was part of an attempt to separate some of the configuration but it made it more complex at the time. I'll remove it.